### ISSUE TITLE
Sparql computed trace : "inherit=false" parameter act as "inherit=true"

### ISSUE THREAD

**Issue by fconil** [ktbs/ktbs#32]

**Comment by pchampin** [comment]
Indeed, it is the *presence* of the inherit parameter which is relevant, not its value.

Still, I agree that this is highly counter-intuitive... But the problem is: which values should be considered as false? A sensible approach would be to follow [ConfigParser.getboolean](https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean).

I still have a problem with the empty string, though... On the one hand, the empty string is traditionally considered as false. On the other hand, I see three arguments in favor of considering it as "true" in this context:
* backward compatibility (currently, it is considered as true);
* visually, the parameter "inherit=" does not look like a negation, rather like a flag that is set (hence true);
* the empty string is not allowed by [ConfigParser.getboolean](https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean), so we are not reusing it in an inconsistent manner.
